device/vxlan/unicast.yaml

```diff
@@ -29,17 +25,29 @@ components:
             type: string
             format: ipv4
 
-    Remote.Vtep.Address:
+    Vxlan.V4Tunnel.DestinationIPMode.Unicast:
+      x-include: 
+      - '#/components/schemas/Vxlan.Tunnel.DestinationIPMode.UnicastBase'
+      description: >-
+        Configuration of Unicast parameters for VxLAN.
       type: object
       properties:
-        choice:
-          description: IPv4 or IPv6 address
-          type: string
-          enum: [ipv4, ipv6]
-          default: ipv4
-        ipv4:
+        remote_vtep_address:
+          description: >-
+            Remote VXLAN Tunnel End Point address
           type: string
           format: ipv4
-        ipv6:
+
+    Vxlan.V6Tunnel.DestinationIPMode.Unicast:
+      x-include: 
+      - '#/components/schemas/Vxlan.Tunnel.DestinationIPMode.UnicastBase'
+      description: >-
+        Configuration of Unicast parameters for VxLAN.
```

Comment:
How about : 
"Configuration of Unicast parameters for an IPv6 VxLAN tunnel"?

Reply:
updated